client/src/modules/journal/journal.service.js

```diff
@@ -180,5 +202,12 @@ function JournalService(Api, AppCache, Filters, Periods, Modal) {
     }).result;
   }
 
+  // updating the posting journal by adding comments in transactions
+  function commentPostingJournal(params) {
+    return service.$http.put(URL.concat('comments'), { 'params' : params })
+      .then(util.unwrapHttpResponse);
```

Comment:
The looks correct :+1:

You actually don't need to inject `util` here.  It is available on the `service` instance.  Like this:

```js
.then(service.util.unwrapHttpResponse);
```

Reply:
Then you can remove the extra `$inject` for util.